src/cinder/gl/ShaderPreprocessor.cpp

```diff
@@ -85,9 +86,9 @@ std::string ShaderPreprocessor::parseDirectives( const std::string &source )
 	
 	// if we don't have a version yet, add the default one
 	if( version.empty() ) {
-#if defined( CINDER_GL_ES )
-		version = "#version " + to_string( mVersion ) + "es\n";
-#else
+#if defined( CINDER_GL_ES_3 )
+		version = "#version " + to_string( mVersion ) + " es\n";
+#elif defined( CINDER_GL_ES_2 )
```

Comment:
I think you're right on the point that es2 doesn't use " es" suffix, but this looks like it doesn't cover desktop anymore. I think the `#elif defined( CINDER_GL_ES_2 )` should just be and `#else`.